transifex.yml

```diff
@@ -0,0 +1,13 @@
+git:
+  filters:
+  - filter_type: file
+    file_format: XLIFF
+    source_language: en_US
+    source_file: damus Localizations/en-US.xcloc/Localized Contents/en-US.xliff
+    translation_files_expression: damus Localizations/<lang>.xcloc/Localized Contents/<lang>.xliff
+
+  settings:
+    pr_branch_name: transifex/<br_unique_id>
+    language_mapping:
+      en_US: en-US
+      es_419: es-419
```

Comment:
Amended a fix on the last commit. Turns out Transifex uses locale codes in the form of `es_419`, for example, but Xcode uses `es-419`. i.e. Transifex uses `_` (underscore) to separate language and region/variant, and Xcode uses `-` (dash). We can use this mapping to convert between the two. It's a pain to keep this list up to date, but not sure if there are alternative options.